### PR TITLE
Run ci checks for Migrate Chargify Source Connector to Low Code

### LINKED ISSUE #28988: chargify
Please use the [connector builder](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview/?_gl=1*13hxonw*_gcl_au*MTI0NTMzNjM3OS4xNjkwODM4MzE0*_ga*MTg2NTc3OTc0Ni4xNjUxNTk0NTQw*_ga_HDBMVFQGBH*MTY5MTAwNDEzMy4yNzIuMC4xNjkxMDA0MTMzLjAuMC4w&_ga=2.197654767.1978878.1690818922-1865779746.1651594540) for this.

### PR DESCRIPTION
**Pull request by sajarin**
## What
Migrating Source Chargify to Low-Code CDK resolves #28988

## How
Created the connector using the connector buider UI, then imported the manifest.yaml to the low code cdk.

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

No breaking changes

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-chargify test report (commit [`2d58228577`](https://github.com/airbytehq/airbyte/pull/29374/commits/2d5822857764429ceb918f4946a8a59efe6e3b92)) - ✅

⏲️  Total pipeline duration: 02mn08s 

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-chargify/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-chargify docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/saj_chargify-low/1691779564/2d5822857764429ceb918f4946a8a59efe6e3b92/source-chargify/0.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:2d5822857764429ceb918f4946a8a59efe6e3b92)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-chargify test
```

File: airbyte-integrations/connectors/source-chargify/source_chargify/schemas/TODO.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# TODO: Define your stream schemas
+Your connector must describe the schema of each stream it can output using [JSONSchema](https://json-schema.org). 
+
+You can describe the schema of your streams using one `.json` file per stream.
+ 
+## Static schemas
+From the `chargify.yaml` configuration file, you read the `.json` files in the `schemas/` directory. You can refer to a schema in your configuration file using the `schema_loader` component's `file_path` field. For example:
+```
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-chargify/source_chargify/schemas/TODO.md:8]: Remove this.

File: airbyte-integrations/connectors/source-chargify/source_chargify/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    title: Chargify Spec
+    type: object
+    required:
+      - api_key
+      - domain
+    additionalProperties: true
+    properties:
+      api_key:
+        type: string
+        title: api_key
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-chargify/source_chargify/manifest.yaml:89]: Can we fix or improve here to use Capitalize text?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-chargify test report (commit [`9e582d3e60`](https://github.com/airbytehq/airbyte/pull/29374/commits/9e582d3e6066b7d3332ae27c5acd2525077d14a0)) - ✅

⏲️  Total pipeline duration: 02mn06s 

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-chargify/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-chargify docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/saj_chargify-low/1691779815/9e582d3e6066b7d3332ae27c5acd2525077d14a0/source-chargify/0.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:9e582d3e6066b7d3332ae27c5acd2525077d14a0)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-chargify test
```

**Comment by octavia-squidington-iii** [comment]
##  source-chargify test report (commit [`fcfaed5e0f`](https://github.com/airbytehq/airbyte/pull/29374/commits/fcfaed5e0fc52d21301366c6db01b8cfa9f40920)) - ✅

⏲️  Total pipeline duration: 01mn37s 

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-chargify/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-chargify docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/saj_chargify-low/1691780480/fcfaed5e0fc52d21301366c6db01b8cfa9f40920/source-chargify/0.3.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:fcfaed5e0fc52d21301366c6db01b8cfa9f40920)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-chargify test
```

**Pull request closed without merging**